lib/socialcast-git-extensions/cli.rb

```diff
@@ -311,6 +311,7 @@ def release
         run_cmd "git pull . #{branch}"
         run_cmd "git push origin HEAD"
         integrate_branch(base_branch, staging_branch)
+        integrate_branch(base_branch, prototype_branch) unless enforce_staging_before_release?
```

Comment:
I don't feel very strongly about it, but my goal was to not disrupt everyone else's git flow.